quantum/matrix.c

```diff
@@ -63,9 +63,9 @@ static const pin_t col_pins[MATRIX_COLS] = MATRIX_COL_PINS;
 #endif
 
 /* matrix state(1:on, 0:off) */
-static matrix_row_t matrix[MATRIX_ROWS];
+__attribute__((weak)) matrix_row_t matrix[MATRIX_ROWS];
```

Comment:
These files shouldn't be modified, and instead, you should be using a custom matrix.

Reply:
I am using a custom matrix, but I wanted to use functions from this file without having a second `matrix` variable linked in. I'll take this change out and copy the functions from this file to my `matrix.c`.